src/main/java/nextstep/subway/common/exception/SubwayExceptionType.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.subway.common.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum SubwayExceptionType {
+
+    INTERNAL_SERVER_ERROR("500", "Server Error"),
```

Comment:
HTTP 응답 코드와 동일한 코드를 사용하실 거라면, 기존에 잘 정의된 상수를 활용해보는 건 어떨까요?
```suggestion
    INTERNAL_SERVER_ERROR(HttpStatus.INTERNAL_SERVER_ERROR.value()), "Server Error"),
```

Reply:
좋은 의견 감사합니다! 기존에 정의된 상수를 사용하면 가독성도 좋아지고, 의미를 좀 더 정확하게 전달할 수 있을 것 같습니다 👍🏼